cmd/runhcs/create-scratch.go

```diff
@@ -52,6 +52,9 @@ var createScratchCommand = cli.Command{
 		// 256MB with boot from vhd supported.
 		opts.MemorySizeInMB = 256
 		opts.VPMemDeviceCount = 1
+		// Default SCSI controller count is 4, we don't need that for this UVM,
```

Comment:
This is a mistake right? Shouldnt we leave the default in the opts as 1 and let someone upgrade if they want?

Reply:
Actually, the default is set to 4 now for LCOW options. But here we don't want to enable 4 controllers so we bring it back to 1. However, this is a good point. We need 4 controllers only when we are not using vpmem devices. So I will make a change to enable 4 SCSI controllers for LCOW only if vpmem is disabled.